options/index.html

```diff
@@ -5,55 +5,44 @@
         <meta charset="UTF-8">
         <link rel="stylesheet" href="options.css" />
         <link rel="stylesheet" href="catblock.css" />
-        <script src="/lib/punycode.min.js"></script>
-        <script src="/lib/jquery.min.js"></script>
-        <script src="/js/port.js"></script>
-        <script src="/js/functions.js"></script>
-        <script src="/js/checkupdates.js"></script>
+        <script src="../lib/punycode.min.js"></script>
+        <script src="../lib/jquery.min.js"></script>
+        <script src="../js/port.js"></script>
+        <script src="../js/functions.js"></script>
+        <script src="../js/checkupdates.js"></script>
```

Comment:
There was a problem loading those scripts under Safari when using only slash at the beginning. This fixes it.